.github/actions/smoke-tests/action.yaml

```diff
@@ -32,6 +38,16 @@ runs:
         path: ${{ github.workspace }}/dist
         key: nginx-ingress-${{ github.run_id }}-${{ github.run_number }}-single
 
+    - name: Ingress type
+      id: ingress-type
+      run: |
+        plus=""
+        if [[ ${{ inputs.image }} == *plus* ]]; then
```

Comment:
I'm still not 100% sure about making decisions about an image based on if the string 'plus' is in the name. @ciarams87 ?

Reply:
Just to clarify this is just to figure out which type of tests to run based on the name of the image. All the plus images have `plus` in the name.

Reply:
Other than hardcoding it elsewhere, there isn't many other good options. The test suite literally does the same thing when judging whether or not it should run the AP tests. I think this is fine.